web/src/components/storage/VolumeGroupEditor.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) [2025] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { _ } from "~/i18n";
+import { sprintf } from "sprintf-js";
+import { configModel } from "~/api/storage/types";
+import { contentDescription } from "~/components/storage/utils/volumeGroup";
+import { useVolumeGroup } from "~/queries/storage/config-model";
+import {
+  DeviceHeader,
+  DeviceMenu,
+  MountPathMenuItem,
+} from "~/components/storage/utils/configEditor";
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Flex,
+  MenuItem,
+  MenuList,
+} from "@patternfly/react-core";
+
+import spacingStyles from "@patternfly/react-styles/css/utilities/Spacing/spacing";
+
+export type VolumeGroupEditorProps = { vg: configModel.VolumeGroup };
+
+const RemoveVgOption = ({ vg }) => {
+  const volumeGroup = useVolumeGroup(vg.name);
+  const drive = volumeGroup.targetDrives[0];
+  const desc = sprintf(_("The logical volumes will become partitions at %s"), drive.name);
```

Comment:
Do we want to do this without asking?

Reply:
Well, it's kind of needed now because we cannot delete the mandatory mount points. So we need to do that in order to delete the VG containing "/". But there is already a note at the card meant to really implement this. https://trello.com/c/RTmm8dzb/1136-storage-ui-delete-volume-group